### PR TITLE
Fix entrypoint

### PR DESCRIPTION
**Pull request by ssz1997**
`format` commands don't exist in cli anymore, so as job services. Delete some entrypoint code.

Solves https://github.com/Alluxio/alluxio/issues/18466

### CODE REVIEW

File: integration/docker/entrypoint.sh [hunk 1]
```diff
@@ -185,29 +185,6 @@ function setup_for_dynamic_non_root {
...
-#######################################
-# Sets the Alluxio ram folder if alluxio top tier is MEM and ram folder isn't explicitly configured.
-# Globals:
-#   ALLUXIO_JAVA_OPTS
-#   ALLUXIO_HOME
-#   ALLUXIO_WORKER_JAVA_OPTS
-# Arguments:
-#   None
-#######################################
-function set_ram_folder_if_needed {
```
Comment (ChunxuTang) [inline integration/docker/entrypoint.sh:197]: Any reasons to remove this function? Is it unused?

**Review by ChunxuTang — APPROVED** [review]

File: integration/docker/entrypoint.sh (hunk 1 above)
Comment (ssz1997) [reply integration/docker/entrypoint.sh:197]: There's no tiered store in dora anymore, so the property key in this function is no longer there and thus the function is useless. And Dora doesn't check for whether the storage is mem/ssd/hdd - it just uses the path.

**Comment by ssz1997** [comment]
alluxio-bot, merge this please.

**Comment by ssz1997** [comment]
alluxio-bot, merge this please.

**Pull request merged**